python/tests/tsutil.py

```diff
@@ -780,10 +780,12 @@ def py_union(tables, other, nodes, record_provenance=True, add_populations=True)
             )
             node_map[other_id] = node_id
     individuals = tables.individuals
+    new_parents = individuals.parents
     for i in range(
         individuals.parents_offset[original_num_individuals], len(individuals.parents)
     ):
-        individuals.parents[i] = ind_map[individuals.parents[i]]
+        new_parents[i] = ind_map[individuals.parents[i]]
+    individuals.parents = new_parents
```

Comment:
doh!!!

Reply:
Gah, that looks like my bad, sorry! I thought this would have been caught by the tests. Also a good argument for allowing assignment in the tables api.